### ISSUE TITLE
Build tutorial is out of date

### ISSUE THREAD

**Issue by ghost** [Innoxia/liliths-throne-public#1735]
**Describe the bug**
The build tutorial recommends Java 8.

Building the game with Java 8 is no longer possible.

**Comment by CognitiveMist** [comment]
Yes, the build tutorial has not been updated since @Tazze overhauled the maven build process in #1684
